torch_dvf/models/pointnet/se.py

```diff
@@ -0,0 +1,120 @@
+import torch
+
+from e3nn.o3 import Irreps
+
+from torch_dvf.models import architectures
+from torch_dvf.nn.gnn import PointCloudPooling
+from torch_dvf.nn.mlp.se import SEMLP
+
+
+"""
+The following class (BalancedIrreps) is copied from "E(3) Steerable GNN" on GitHub (https://github.com/RobDHess/Steerable-E3-GNN).
+"""
+
+def BalancedIrreps(lmax, vec_dim, sh_type=True):
```

Comment:
Add hint typing here. Can we also make it to be a class that inherits from Irreps, since basically this is just constructor in a function.

Reply:
I copy & pasted this verbatim from [here](https://github.com/RobDHess/Steerable-E3-GNN). I think it is better to either leave the code in its original form or to completely rewrite it.